schema/blocks/confirmation.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "title": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "number": {
+        "type": "string"
+      },
+      "type": {
+        "type": "string",
+        "pattern": "^Confirmation$"
+      },
+      "questions": {
+        "type": "array",
+        "items": {
+          "oneOf": [
+            {
+              "$ref": "../questions/general.json#/question"
+            },
+            {
+              "$ref": "../questions/date_range.json#/question"
+            },
+            {
+              "$ref": "../questions/repeating_answer.json#/question"
+            },
+            {
+              "$ref": "../questions/relationship.json#/question"
```

Comment:
Although this is possible, we only ever use the general question type in a confirmation block, so we should restrict it here to ensure no other question types are used. 

We will create some proper structure for the confirmation block when the confirmation/summary pattern is finalised.